src/containers/MapContext/reducer.js

```diff
@@ -39,6 +41,7 @@ export default (state, action) => {
     case SET_VALUES:
       return {
         ...state,
+        loading: false,
```

Comment:
What if `setLocationAction` is called, but `setValuesAction` isn't? There isn't an indicator anywhere in the code that tells me that I need to do that in order to set the `loading` prop. Also, you can set the location just by calling `setValuesAction`, which bypasses the `loading` prop altogether.
I think it would be better to have a separate action to set and unset the `loading` prop instead of hiding it in another action.

Reply:
I agree. Is better to have it separate.